services/ui-src/src/components/cards/EntityStepCard.test.tsx

```diff
@@ -179,132 +179,134 @@ const NoOpenAddEditEntityFunction = (
   />
 );
 
-describe("Test Completed EntityCard", () => {
-  beforeEach(() => {
-    render(GenericEntityTypeEntityCardComponent);
+describe("<EntityCard />", () => {
+  describe("Completed EntityCard", () => {
+    describe("Test Completed EntityCard", () => {
```

Comment:
same q about multiple nested describes

Reply:
For this one, it's triple `describe` because I wanted the printout to read more clearly which accessibility test it's for.

```
<EntityCard />
  Completed EntityCard
    Test Completed EntityCard
    Accessibilty
  Unfinished EntityCard
    Test Unfinished EntityCard
    Accessibilty
```

instead of

```
<EntityCard />
  Test Completed EntityCard
  Accessibilty
  Test Unfinished EntityCard
  Accessibilty
```

Might be worth it to try to deduce the component name and have it print out with `<ComponentName /> Accessibility` but it wouldn't cover this scenario where we're testing the accessibility of the component in different states.